lib/web_ui/dev/integration_tests_manager.dart

```diff
@@ -200,6 +230,8 @@ class IntegrationTestsManager {
         'web-server',
         '--profile',
         '--browser-name=$_browser',
+        if (isLuci) '--chrome-binary=${chromeExecutableForLUCI()}',
+        if (isLuci) '--headless',
```

Comment:
Why headless only on LUCI? This will lead to different test results (e.g. clipboard and screenshots behave differently).

Reply:
As we discussed last week, current bots do not have the ability to run in no-headless mode.

I'm discussing with infra teams other options. 

For now our best option would be having 4-5 critical e2e tests running on MacOs on Chrome.

Reply:
What I mean, if we are going to run in headless on LUCI, we should run headless everywhere else, no?